package-lock.json

```diff
@@ -1,8 +1,3123 @@
 {
   "name": "r7insight_node",
-  "version": "3.1.0",
-  "lockfileVersion": 1,
+  "version": "3.1.1",
+  "lockfileVersion": 2,
```

Comment:
updated package lock after `npm install`

this should be completely backwards compatible, you can see no packages are changed, just new format is added as according to npm docs:
```
2: The lockfile version used by npm v7, which is backwards compatible to v1 lockfiles.
```
https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion

Reply:
`This version of npm is compatible with lockfileversion@1, but package-lock.json was generated for lockfileVersion@2`

You sure about that?

Reply:
It _is_ only a warning so it might be fine. But just make sure to check :)

Reply:
npm v5 works locally, and i'm taking npm's word about backwards-compat